{{cookiecutter.slug}}/backend/{{cookiecutter.slug}}/common_settings.py

```diff
@@ -0,0 +1,43 @@
+INSTALLED_APPS = [
```

Comment:
I take it the idea behind this file is a shared settings file for development and all deployment variants?
Good idea, maybe add some comment that developers should not add sensitive information here.